apps/grc-platform/backend/cmd/backfill-evidence-actors/main.go

```
@@ -0,0 +1,248 @@
+	if err != nil {
+		fatal("query risk_evidence.created_by: %v", err)
+	}
+	var actors []string
+	for rows.Next() {
+		var a string
+		if err := rows.Scan(&a); err != nil {
+			fatal("scan created_by: %v", err)
+		}
+		actors = append(actors, a)
+	}
+	rows.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rows.Close` is not checked

(errcheck)